packages/web-components/fast-foundation/src/listbox/listbox.template.ts

```diff
@@ -13,6 +13,7 @@ export const listboxTemplate: FoundationElementTemplate<ViewTemplate<ListboxElem
 ) => html`
     <template
         aria-activedescendant="${x => x.ariaActiveDescendant}"
+        aria-multiselectable="${x => x.ariaMultiselectable}"
```

Comment:
The IDL (listed elsewhere) is [`ariaMultiSelectable`](https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaMultiSelectable), it seems like we'll want to map to that.